librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -1453,3 +1453,100 @@ commands:
               - name: class_name
                 type: RZ_CMD_ARG_TYPE_STRING
                 optional: true
+  - name: a8
+    summary: Analyze bytes
+    cname: analyze_bytes
+    args:
+      - name: hexpairs
+        type: RZ_CMD_ARG_TYPE_STRING
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_JSON
+  - name: aO
+    summary: Analyze N instructions in M bytes
+    subcommands:
+      - name: aO
+        summary: Analyze next N bytes as instructions
+        cname: analyze_n_bytes
```

Comment:
this, aOe, aOd and aOs all works on bytes and essentially just change the block size.
I propose to *remove* the n_bytes argument from these command handlers and just rely on core->blocksize by default. Also, change the description of these commands to say "analyze ... next block as instructions".
Users can easily use `aO @!10` to set the block size to 10, so we don't need these command handlers to do the same work.

cc @wargio